core/primitives-core/src/config.rs

```diff
@@ -419,6 +480,118 @@ pub fn default_read_cached_trie_node() -> Gas {
+            ),
+            storage_iter_create_prefix_byte: read_parameter(
+                params,
+                Parameter::WasmStorageIterCreatePrefixByte,
+            ),
+            storage_iter_create_range_base: read_parameter(
+                params,
+                Parameter::WasmStorageIterCreateRangeBase,
+            ),
+            storage_iter_create_from_byte: read_parameter(
+                params,
+                Parameter::WasmStorageIterCreateFromByte,
+            ),
+            storage_iter_create_to_byte: read_parameter(
+                params,
+                Parameter::WasmStorageIterCreateToByte,
+            ),
+            storage_iter_next_base: read_parameter(params, Parameter::WasmStorageIterNextBase),
+            storage_iter_next_key_byte: read_parameter(
+                params,
+                Parameter::WasmStorageIterNextKeyByte,
+            ),
+            storage_iter_next_value_byte: read_parameter(
+                params,
+                Parameter::WasmStorageIterNextValueByte,
+            ),
+            touching_trie_node: read_parameter(params, Parameter::WasmTouchingTrieNode),
+            read_cached_trie_node: read_parameter(params, Parameter::WasmReadCachedTrieNode),
+            promise_and_base: read_parameter(params, Parameter::WasmPromiseAndBase),
+            promise_and_per_promise: read_parameter(params, Parameter::WasmPromiseAndPerPromise),
+            promise_return: read_parameter(params, Parameter::WasmPromiseReturn),
+            validator_stake_base: read_parameter(params, Parameter::WasmValidatorStakeBase),
+            validator_total_stake_base: read_parameter(
+                params,
+                Parameter::WasmValidatorTotalStakeBase,
+            ),
+            #[cfg(feature = "protocol_feature_alt_bn128")]
+            alt_bn128_g1_multiexp_base: read_parameter(
+                params,
+                Parameter::WasmAltBn128G1MultiexpBase,
+            ),
+            #[cfg(feature = "protocol_feature_alt_bn128")]
+            alt_bn128_g1_multiexp_element: read_parameter(
+                params,
+                Parameter::WasmAltBn128G1MultiexpElement,
+            ),
+            #[cfg(feature = "protocol_feature_alt_bn128")]
+            alt_bn128_pairing_check_base: read_parameter(
+                params,
+                Parameter::WasmAltBn128PairingCheckBase,
+            ),
+            #[cfg(feature = "protocol_feature_alt_bn128")]
+            alt_bn128_pairing_check_element: read_parameter(
+                params,
+                Parameter::WasmAltBn128PairingCheckElement,
+            ),
+            #[cfg(feature = "protocol_feature_alt_bn128")]
+            alt_bn128_g1_sum_base: read_parameter(params, Parameter::WasmAltBn128G1SumBase),
+            #[cfg(feature = "protocol_feature_alt_bn128")]
+            alt_bn128_g1_sum_element: read_parameter(params, Parameter::WasmAltBn128G1SumElement),
+        }
+    }
+
```

Comment:
let's maybe move these bulky statements to the `parameter` module?

Reply:
My latest commit moves all new constructors to `parameter_table` but left other constructors where they are. Not sure if this is what you had in mind.